README.md

```diff
@@ -1,2 +1,26 @@
 # Functional.CQS
-Defines CQS-style contracts and provides AOP facilities over those contracts using [Functional](https://github.com/JohannesMoersch/Functional) primitives
+
+Defines [CQS-style contracts](https://github.com/RyanMarcotte/Functional.CQS/tree/master/src/Functional.CQS) using [Functional primitives](https://github.com/JohannesMoersch/Functional):
+- [`IQueryParameters<TResult>`](src/Functional.CQS/IQueryParameters.cs), a marker interface for query parameter objects that associates the query parameter object with the type of result that is returned when invoking the query handler
+- [`IQueryHandler<TQuery, TResult>`](src/Functional.CQS/IQueryHandler.cs), for synchronous query handling
+- [`IAsyncQueryHandler<TQuery, TResult>`](src/Functional.CQS/IAsyncQueryHandler.cs), for asynchronous query handling
+- [`ICommandParameters<TError>`](src/Functional.CQS/ICommandParameters.cs), a marker interface for command parameter objects that associates the command parameter object with the type of error that can potentially occur when invoking the command handler
+- [`ICommandHandler<TCommand, TError>`](src/Functional.CQS/ICommandHandler.cs), for synchronous command handling
+- [`IAsyncCommandHandler<TCommand, TError>`](src/Functional.CQS/IAsyncCommandHandler.cs), for asynchronous command handling
+
+# Functional.CQS.AOP
+
+`Functional.CQS.AOP` is a set of NuGet packages that provide decoration facilities over implementations of `Functional.CQS` contracts (listed above), specifically for [aspect-oriented programming](https://en.wikipedia.org/wiki/Aspect-oriented_programming) / applying [cross-cutting concerns](https://stackoverflow.com/questions/23700540/cross-cutting-concern-example).
+
+[What is CQS?  Why use CQS?  Why use Functional.CQS.AOP?](docs/cqs_primer.md)
```

Comment:
Does it make sense to have this link be in its own section _(eg. just put a `## Background` header, or some similar title, above this link)_? It feels like this link should stand out a little more, and I don't think that's achieved by having it be part of the `Functional.CQS.AOP` library's description.

Reply:
It makes to move a few things around.  Will divide the existing `cqs_primer` page into two.

Reply:
Done